Monika-After-Story/MonikaModDev
# Menu transitions

This adds transforms to some of our screens. They are basically used as transitions (on showing/hiding the screens). I didn't notice any performance issues, but just in case this does respect user preferences, so people with toggled off animation won't notice any difference.

### Testing:
- verify the transforms and "transitions" overall look good
- verify no new performance issues nor other bugs
- verify the talk menu always slides in when you open it, slides out when you close it, **but always dissolves out when open a submenu** (bookmarks for example)
- verify that the twopane screen dissolves only when you first time open it, or when you close it, **but not when you change categories**
- verify that the base dlg menu (the screen that is used by the `menu` statement in dlgs) is being shown/hidden with dissolve
- verify the music menu slides in when you first time open it, and slides out when you close it, **but no transition happens when you change pages**
- verify that the calendar is being shown/hidden with dissolve
- verify the screens don't use transforms if you have `persistent._mas_disable_animations` set to `False`
- verify you can't use other menus in the game menu (via hotkeys)
